### ISSUE TITLE
netxml-ups driver missing from current builds

### ISSUE THREAD

**Issue by sondercoder** [hassio-addons/addon-nut#125]
# Problem/Motivation
After update, no longer able to monitor UPS systems

## Expected behavior
Monitoring continues to work with previously working UPS driver

## Actual behavior
Process fails to run as driver is not found.

## Steps to reproduce
Create configuration using the netxml-ups driver:

Current Config:

```
users:
  - username: admin
    password: ********
    instcmds:
      - all
    actions: []
devices:
  - name: EATON_9130
    driver: netxml-ups
    port: 'http://192.168.*.*'
    config: []
  - name: EATON_5130
    driver: netxml-ups
    port: 'http://192.168.*.*'
    config: []
mode: netserver
shutdown_host: 'false'
```

Log Output:
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] nut: applying... 
[fix-attrs.d] nut: exited 0.
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: Network UPS Tools
 Manage battery backup (UPS) devices
-----------------------------------------------------------
 Add-on version: 0.6.2
 You are running the latest version of this add-on.
 System: Raspbian GNU/Linux 10 (buster)  (armv7 / raspberrypi4)
 Home Assistant Core: 2021.4.6
 Home Assistant Supervisor: 2021.04.3
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] nut.sh: executing... 
[10:31:48] INFO: Setting mode to netserver...
[10:31:48] INFO: Generating /etc/nut/upsd.users...
[10:31:48] INFO: Configuring user: admin
[10:31:48] INFO: Configuring Device named EATON_9130...
[10:31:48] INFO: Configuring Device named EATON_5130...
[10:31:48] INFO: Starting the UPS drivers...
Network UPS Tools - UPS driver controller 2.7.4
Can't start /lib/nut/netxml-ups: No such file or directory
[cont-init.d] nut.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.

```

## Proposed changes

Include netxml-ups in the included drivers

**Comment by sinclairpaul** [comment]
Fixed in #126

**Comment by sinclairpaul** [comment]
Fixed in v0.7.0
